gradle_exporter.py

```diff
@@ -0,0 +1,123 @@
+import os
+import json
+import aiohttp
+from aiohttp import web
+import asyncio
+import logging
+import argparse
+
+logging.basicConfig(level=os.getenv("LOG_LEVEL", logging.DEBUG), format='%(asctime)s - %(levelname)s - %(message)s')
```

Comment:
It is better to use `loggin.INFO` by default

Reply:
Fixed